bindings/c/test/mako/mako.cpp

```diff
@@ -1575,7 +1501,7 @@ void printReport(Arguments const& args,
 					} else {
 						fmt::fprintf(fp, ",");
 					}
-					fmt::fprintf(fp, "\"%s\": %lu", getOpName(op), lat_min);
+					fmt::fprintf(fp, "\"%s\": %lu", op_name, lat_min);
```

Comment:
if you only use `op_name` once, you can just replace it's usage with `getOpName(op)`